team-agreements/definition-of-ready/readme.md

```diff
@@ -1,8 +1,30 @@
-# Team Definition Of Ready Agreement
+# Definition of Ready
 
-## Content TODO's
+At the time the development team picks a user story from the top of the backlog to estimate during the Sprint planning, they consider that the user story has a certain level of detail that helps them to give an accurate estimate of completion of the user story within the sprint. This level of information is what makes the story to be Ready to be developed in the sprint.
 
-- Define criteria for a story to be ready to pull into a sprint.
-  - Examples
-    - Has estimate
-    - Team understands implementation/design
+  `In case a user story is not Ready in the beginning of the Sprint it increases the chance that your story will not be done at the end of this print.`
+
+## What it is
+
+Definition of Ready is the agreement made by the scrum team around how complete a user story should be to be selected as candidate for estimation in the sprint planning.
+
+It can be understood as a checklist that helps the Product Owner to ensure that the user story he wrote contains all the necessary and the enough content to the scrum team.
+
+### Examples of ready checklist items:
+
+- The description had enough details and input values mandatory in the implementation.
+- The user story contained a clear and a complete acceptance criteria.
+- The user story addressed the business view
+- The acceptance criteria could be measured
+
+## Who writes it
+
+The ready checklist can be written by a Product Owner in agreement with the development team and the scrum master.
+
+## When a Definition of Ready needs an update
+
+You can update or change the definition of ready anytime the scrum team observes that there are missing information in the user stories that recurrently impact the good evolution of the planning and that recurrent information was not in the checklist.
+
+## What should be avoided
+
+It is desirable that the ready checklist can apply to two or more user stories. We discourage you of including in your ready checklist any detail that make the ready checklist to become one of a kind and too specific for a single user story. This may lead to a 1:1 relationship between the user story and the checklist, what may delay the process of writing good user stories.

```

Comment:
Suggestion for reword:

The ready checklist should contain items that apply broadly.  Don't include items that only apply to one or two user stories.

Reply:
Done. Thanks.